internal/pkg/cli/project_init.go

```diff
@@ -151,6 +156,62 @@ func (o *initProjectOpts) Execute() error {
 		return err
 	}
 	o.prog.Stop(log.Ssuccessf(fmtDeployProjectComplete, color.HighlightUserInput(o.ProjectName)))
+
+	return o.projectStore.CreateProject(&archer.Project{
+		AccountID: caller.Account,
+		Name:      o.ProjectName,
+		Domain:    o.DomainName,
+	})
+}
+
+func (o *initProjectOpts) validateProject(projectName string) error {
+	if err := validateProjectName(projectName); err != nil {
+		return err
+	}
+	// Alert users if the project already exists.
+	if _, err := o.projectStore.GetProject(projectName); err != nil {
+		var noSuchProjectErr *store.ErrNoSuchProject
+		if errors.As(err, &noSuchProjectErr) {
+			return nil
+		}
+		return err
+	}
+	confirm, err := o.prompt.Confirm(fmt.Sprintf("Project %s already exists, would you like to use it?", projectName), "", prompt.WithTrueDefault())
+	if err != nil {
+		return fmt.Errorf("prompt to confirm using existing project: %w", err)
+	}
+	if !confirm {
+		return fmt.Errorf("project init cancelled - no changes made")
+	}
+
+	return nil
+}
+
+func (o *initProjectOpts) validateDomain(domainName string) error {
+	domainExist := false
```

Comment:
What do you think of putting this functionality in the `route53` package instead?

Reply:
:+1: good idea